insights/parsers/systemctl_show.py

```diff
@@ -105,3 +105,100 @@ class SystemctlShowMariaDB(SystemctlShow):
         network.target -.mount systemd-journald.socket tmp.mount basic.target syslog.target system.slice
     """
     pass
+
+
+@parser('systemctl_pulp_workers')
+class SystemctlShowPulpWorkers(SystemctlShow):
```

Comment:
@TripleJQK this module has subheadings at the top of the file, please add the new classes to the subheadings.